src/components/Home/EventDetail/index.js

```diff
@@ -1,29 +1,143 @@
+/* eslint-disable camelcase */
 import React, { Component } from 'react';
 import {
-  StyleSheet, Text, View, TouchableOpacity,
+  Text, ScrollView, TouchableOpacity, Image, View, ActivityIndicator,
 } from 'react-native';
-
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    justifyContent: 'center',
-    alignItems: 'center',
-    backgroundColor: '#F5FCFF',
-  },
-});
+import { connect } from 'react-redux';
+import MapView, { Marker } from 'react-native-maps';
+import FontAwesome from 'react-native-vector-icons/FontAwesome';
+import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
+import Entypo from 'react-native-vector-icons/Entypo';
+import AntDesign from 'react-native-vector-icons/AntDesign';
+import moment from 'moment';
+import { loadEventDetailDispatch } from 'datalayer/actions/event.action';
+import Dialog from 'utils/errorDialog';
+import { isEmptyObject } from 'utils/object';
+import styles from './index.styles';
 
 class EventDetail extends Component {
+  componentDidMount() {
+    const { navigation, loadEventDetailDispatch } = this.props;
+    const eventId = navigation.getParam('eventId');
+    loadEventDetailDispatch(eventId)
+      .then(res => {
+        if (!res.success) {
+          Dialog.show(res.error);
+        }
+      });
+  }
+
+  goToGuestsList = () => {
+
+  }
+
   render() {
-    const { navigation } = this.props;
+    const { eventDetail } = this.props;
+    if (isEmptyObject(eventDetail)) {
```

Comment:
Why check with `isEmptyObject`?  Will there be undefined or null?

Reply:
Because I set the *eventDetail* reducer with initial state is { }

Reply:
Ok